nvme-print-stdout.c

```diff
@@ -996,20 +992,20 @@ static void stdout_supported_cap_config_log(struct nvme_supported_cap_config_lis
 					cap->cap_config_desc[i].egcd[j].end_est)));
 			egsets = le16_to_cpu(cap->cap_config_desc[i].egcd[j].egsets);
 			printf("Number of NVM Sets: %u\n", egsets);
-			for(k = 0; k < egsets; k++) {
+			for (k = 0; k < egsets; k++) {
 				printf("NVM Set %d Identifier: %u\n", i,
 					le16_to_cpu(cap->cap_config_desc[i].egcd[j].nvmsetid[k]));
 			}
-			chan_desc = (struct nvme_end_grp_chan_desc *) \
-					((cap->cap_config_desc[i].egcd[j].nvmsetid[0]) * (sizeof(__u16)*egsets));
+			chan_desc = (struct nvme_end_grp_chan_desc *)
+			    (sizeof(__u16) * egsets * cap->cap_config_desc[i].egcd[j].nvmsetid[0]);
```

Comment:
This part should be in the next patch. It's not a white space change thus it should get a proper commit message with it.